GpsDataAnalyzer/test/test_deviation_calculator.py

```diff
@@ -11,163 +11,176 @@
-                 datetime(2020, 7, 22, 15, 49, 8, tzinfo=timezone.utc)]
-    deviation_list = [0.0, 0.0, 0.0, 0.0, 0.0]
-    speed_differentials = [1.0, 0.0, 0.0, 0.0, 0.0]
-    altitude_differentials = [.3, 0.0, 0.0, 0.0, 0.0]
-    set1_time_list = [datetime(2020, 7, 22, 15, 49, 4, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 5, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 6, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 7, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 8, tzinfo=timezone.utc)]
-    set2_time_list = [datetime(2020, 7, 22, 15, 49, 1, 626803, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 2, 626803, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 3, 626803, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 4, 626803, tzinfo=timezone.utc),
-                      datetime(2020, 7, 22, 15, 49, 5, 626803, tzinfo=timezone.utc)]
-    deviations_dataframe = pd.DataFrame({"Common Timestamp": time_list,
-                                         "Deviations": deviation_list,
-                                         "Speed Differentials": speed_differentials,
-                                         "Altitude Differentials": altitude_differentials,
-                                         "Set 1 Timestamp": set1_time_list,
-                                         "Set 2 Timestamp": set2_time_list})
-    calculator = DataSetDeviationCalculator(self.watch_data_set, self.simulator_data_set)
-
-    result = calculator.get_deviation_dataframe()
-
-    pd.testing.assert_frame_equal(deviations_dataframe, result)
-
-  def test_get_deviation_dataframe_no_lineup(self):
-    deviations_dataframe = pd.DataFrame({"Common Timestamp": [],
-                                         "Deviations": [],
-                                         "Speed Differentials": [],
-                                         "Altitude Differentials": [],
-                                         "Set 1 Timestamp": [],
-                                         "Set 2 Timestamp": []})
-    for data_point in self.simulator_data_set.gps_data_list:
-      data_point.time = data_point.time + timedelta(seconds=100)
-    calculator = DataSetDeviationCalculator(self.watch_data_set, self.simulator_data_set)
-
-    result = calculator.get_deviation_dataframe()
-
-    pd.testing.assert_frame_equal(deviations_dataframe, result)
+	def setUp(self):
+		watch_metadata = GpsMetaData(device='sampleDevice',
+									 identifier='sampleId',
+									 manufacturer='sampleManufacturer',
+									 model='sampleModel',
+									 start_time=datetime(2020, 7, 7, 18,45,47,5000, tzinfo=timezone.utc),
+									 end_time=datetime(2020,7,7,19,8,1,318000,tzinfo=timezone.utc))
+		watch_data_list = [GpsData(latitude= 37.31013773,
+									longitude= -122.0314044,
+									altitude= 44.2,
```

Comment:
Please change all the lat/lon/alt to a random one, for example: `Latitude: -48.31711, Longitude: 163.22660, Altitude: 2.26`